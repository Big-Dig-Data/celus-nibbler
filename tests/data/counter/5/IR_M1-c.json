{
  "Report_Header": {
    "Created": "2022-11-01T00:00:00Z",
    "Created_By": "MyProvider",
    "Customer_ID": "11111",
    "Report_ID": "IR_M1",
    "Release": "5",
    "Report_Name": "Multimedia Item Requests",
    "Institution_Name": "MyUniversity",
    "Institution_ID": [
      {
        "Type": "Proprietary",
        "Value": "MyProvider:11111"
      }
    ],
    "Report_Filters": [
      {
        "Name": "Platform",
        "Value": "MyPlatform"
      },
      {
        "Name": "Begin_Date",
        "Value": "2022-10-01"
      },
      {
        "Name": "End_Date",
        "Value": "2022-10-31"
      },
      {
        "Name": "Metric_Type",
        "Value": "Total_Item_Requests"
      },
      {
        "Name": "Data_Type",
        "Value": "Multimedia"
      },
      {
        "Name": "Access_Method",
        "Value": "Regular"
      }
    ]
  },
  "Report_Items": [
    {
      "Item": "Item1",
      "Item_ID": [
        {
          "Type": "Proprietary",
          "Value": "MyProvider:1111111111"
        }
      ],
      "Platform": "MyPlatform",
      "Publisher": "MyPublisher",
      "Performance": [
        {
          "Period": {
            "Begin_Date": "2022-10-01",
            "End_Date": "2022-10-31"
          },
          "Instance": [
            {
              "Metric_Type": "Total_Item_Requests",
              "Count": 16
            }
          ]
        }
      ]
    },
    {
      "Item": "Item2",
      "Item_ID": [
        {
          "Type": "Proprietary",
          "Value": "MyProvider:2222222222"
        }
      ],
      "Platform": "MyPlatform",
      "Publisher": "MyPublisher",
      "Performance": [
        {
          "Period": {
            "Begin_Date": "2022-10-01",
            "End_Date": "2022-10-31"
          },
          "Instance": [
            {
              "Metric_Type": "Total_Item_Requests",
              "Count": 3
            }
          ]
        }
      ]
    },
    {
      "Item": "Item3",
      "Item_ID": [
        {
          "Type": "Proprietary",
          "Value": "MyProvider:3333333333"
        }
      ],
      "Platform": "MyPlatform",
      "Publisher": "MyPublisher",
      "Performance": [
        {
          "Period": {
            "Begin_Date": "2022-10-01",
            "End_Date": "2022-10-31"
          },
          "Instance": [
            {
              "Metric_Type": "Total_Item_Requests",
              "Count": 1
            }
          ]
        }
      ]
    },
    {
      "Item": "Item4",
      "Item_ID": [
        {
          "Type": "Proprietary",
          "Value": "MyProvider:4444444444"
        }
      ],
      "Platform": "MyPlatform",
      "Publisher": "MyPublisher",
      "Performance": [
        {
          "Period": {
            "Begin_Date": "2022-10-01",
            "End_Date": "2022-10-31"
          },
          "Instance": [
            {
              "Metric_Type": "Total_Item_Requests",
              "Count": 3
            }
          ]
        }
      ]
    },
    {
      "Item": "Item5",
      "Item_ID": [
        {
          "Type": "Proprietary",
          "Value": "MyProvider:5555555555"
        }
      ],
      "Platform": "MyPlatform",
      "Publisher": "MyPublisher",
      "Performance": [
        {
          "Period": {
            "Begin_Date": "2022-10-01",
            "End_Date": "2022-10-31"
          },
          "Instance": [
            {
              "Metric_Type": "Total_Item_Requests",
              "Count": 1
            }
          ]
        }
      ]
    }
  ]
}
