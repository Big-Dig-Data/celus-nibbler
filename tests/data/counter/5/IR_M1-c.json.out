"2022-10-01","2022-10-31","","","","Item1","Proprietary:MyProvider:1111111111","Platform:MyPlatform|Publisher:MyPublisher","Total_Item_Requests","16"
"2022-10-01","2022-10-31","","","","Item2","Proprietary:MyProvider:2222222222","Platform:MyPlatform|Publisher:MyPublisher","Total_Item_Requests","3"
"2022-10-01","2022-10-31","","","","Item3","Proprietary:MyProvider:3333333333","Platform:MyPlatform|Publisher:MyPublisher","Total_Item_Requests","1"
"2022-10-01","2022-10-31","","","","Item4","Proprietary:MyProvider:4444444444","Platform:MyPlatform|Publisher:MyPublisher","Total_Item_Requests","3"
"2022-10-01","2022-10-31","","","","Item5","Proprietary:MyProvider:5555555555","Platform:MyPlatform|Publisher:MyPublisher","Total_Item_Requests","1"
